From 2e3616c01f3a78cd2ef1a013713d9b6c2bf8ea79 Mon Sep 17 00:00:00 2001
From: Zumeng Chen <zumeng.chen@windriver.com>
Date: Wed, 8 Nov 2017 14:35:24 +0800
Subject: [PATCH 33/35] ti-st: remove MODULE_DEVICE_TABLE to avoid be
 installed when boot time

st_drv should be installed manually when need, so remove the module table
of definition.

Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
(cherry picked from commit 94730f279cd227cd3cbfa8419cbcaa74e3eec8c8)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/misc/ti-st/st_kim.c |    1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/misc/ti-st/st_kim.c b/drivers/misc/ti-st/st_kim.c
index a67d59b..7b3e90e 100644
--- a/drivers/misc/ti-st/st_kim.c
+++ b/drivers/misc/ti-st/st_kim.c
@@ -742,7 +742,6 @@ static int kim_list_open(struct inode *i, struct file *f)
 	},
 	{}
 };
-MODULE_DEVICE_TABLE(of, kim_of_match);
 
 static struct ti_st_plat_data *get_platform_data(struct device *dev)
 {
-- 
1.7.9.5

